pkg/apis/cluster/v1alpha1/nodegroup_types.go

```diff
@@ -11,7 +11,27 @@ import (
 type NodeGroupSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
-	Name string `json:"name"`
+	Name        string   `json:"name"`
+	IAMPolicies []Policy `json:"iamPolicies"`
```

Comment:
Can we bump the version, or make this optional.

Reply:
It should be optional as-is. If you don't specify the policies, it becomes an empty list which is left out of the template via an `if` condition.

Reply:
I think Aaron's point is about making it optional in Spec of CRD.
`json:"iamPolicies,omitempty"`

Reply:
agreed this should be optional